hard_reload.py

```diff
@@ -66,8 +60,8 @@ def hard_reload(module_name):
 def show():
     w = HardReload(get_parent())
     w.move_window()
-    w.exec_()
-
+    w.exec()
```

Comment:
question:

Is this compatible with PySide?
If I remember correctly, exec was a reserved keyword on python2 therefore qt developers renamed the method.

Is there any win by renaming? I believe the underscore version has been kept on newer versions for backward compatibility.

Reply:
i don't know
exec_ still works but throws a warning in later versions